Common/Data/Models/MobData.cs

```diff
@@ -6,4 +6,25 @@ public class MobData
 {
 	public int NetId { get; set; }
 	public List<MobEntry> Entries { get; set; }
+}
+
+public class MobEntry
```

Comment:
Note: All of these, to my knowledge, can be made into `record`s without a difference in functionality or readability & less code.

Reply:
Records or classes both work yes. They are functionally the same other than the readonly capabilities AFAIK